source/documentation/upgrade_guide/topics/installing_virtual_product_hypervisors_from_local_repository.adoc

```diff
@@ -0,0 +1,88 @@
+[id="installing_virtual_product_hypervisors_from_local_repository_{context}"]
+= Installing {virt-product-shortname} hypervisors from a local repository
+
+If your system uses a private {virt-product-fullname} ({virt-product-shortname}) environment, but without Red Hat Satellite, you might need to install {virt-product-shortname} hypervisors (RHV-H) from a repository hosted on a local RHEL system instead of the Red Hat hosted Content Delivery Network (CDN).
+
+.Procedure
+. On the system hosting the offline repository, create a file `/etc/yum/repos.d/rhvh-mirror.repo` similar to:
++
+[options="nowrap" subs="normal"]
+----
+[rhvh-4-for-rhel-8-x86_64-rpms]
+name = Red Hat Virtualization Host for RHEL 8 x86_64 (RPMs)
+baseurl = https://cdn.redhat.com/content/dist/layered/rhel8/x86_64/rhvh/4/os
+enabled = 0
+gpgcheck = 1
+gpgkey = file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
+sslverify = 1
+sslcacert = /etc/rhsm/ca/redhat-uep.pem
+metadata_expire = 86400
+enabled_metadata = 1
+sslclientcert =
+sslclientkey =
+----
++
+You need to populate the `sslclientcert` and `sslclientkey` fields with full pathnames to the correct files containing the appropriate certificate and key. The `/etc/pki/entitlement` directory contains one or more pair of certificate and key files, but only one pair contains the RHV-H entitlement you need.
```

Comment:
need > must. IBM says to use "need" only if "must" is too restrictive. https://www.ibm.com/docs/en/ibm-style?topic=word-usage#n